docs/modules/plugins/pages/plugin-web-app-playwright.adoc

```diff
@@ -429,3 +429,109 @@ When I select element located by `id(uploadfile)` and upload file `/folder/file_
 ----
 
 include::plugins:partial$ui-scrolling-steps.adoc[]
+
+=== Request validation steps
+:enable-collecting-note: This step requires requests collecting to be turned on. It can be done in properties via "playwright.requests-collecting-enabled" or by switching on "@collectRequests" meta tag at the story or scenario level.
```

Comment:
```suggestion
:enable-collecting-note: This step requires requests collecting to be turned on. It can be done in properties via "proxy.recording.enabled" or by switching on "@proxy" meta tag at the story or scenario level.
```
?